include/hobbes/net.H

```diff
@@ -324,7 +324,7 @@ _HNET_DEFINE_PRIMTYS(int32_t,  "int");
 _HNET_DEFINE_PRIMTYS(uint32_t, "int");
 _HNET_DEFINE_PRIMTYS(int64_t,  "long");
 _HNET_DEFINE_PRIMTYS(uint64_t, "long");
-_HNET_DEFINE_PRIMTYS(size_t,   "long");
+//_HNET_DEFINE_PRIMTYS(size_t,   "long");
```

Comment:
It must be down to compiler differences we have.  I don't have a problem at all with clock_gettime on OSX for example.

You and Baiyan are right that we should get it working with standard clang builds, but also it should work with whatever they're packaging with XCode (since I would guess that's probably more common on OSX).